README.md

```diff
@@ -82,7 +83,10 @@ when a real user uses it.
   - [`getByTestId`](#getbytestid)
   - [`wait`](#wait)
   - [`waitForElement`](#waitforelement)
+  - [`waitForDomChange`](#waitfordomchange)
   - [`fireEvent`](#fireevent)
+    - [`fireEvent[eventName]`](#fireeventeventname)
+    - [`getNodeText`](#getnodetext)
```

Comment:
Should the indentation be less on this line? getNodeText looks independent of fireEvent.

Reply:
Yep, let's remove one of the `#` from `getNodeText` 👍 Do you mind doing that @themostcolm?

Reply:
Got it.